src/main/kotlin/uk/gov/justice/digital/hmpps/visitallocationapi/model/entity/projections/NegativePrisonerBalance.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.visitallocationapi.model.entity.projections
+
+import uk.gov.justice.digital.hmpps.visitallocationapi.enums.NegativeVisitOrderType
+
+interface NegativePrisonerBalance {
+  val type: NegativeVisitOrderType
+  val balance: Int
```

Comment:
can be UInt as needs to be >=0. but not sure if JPA supports it / and if worth it